core/arch/arm/tee/entry_fast.c

```diff
@@ -64,6 +64,7 @@ static void tee_entry_fastcall_l2cc_mutex(struct thread_smc_args *args)
 
 static void tee_entry_exchange_capabilities(struct thread_smc_args *args)
 {
+	bool res_shm_en = IS_ENABLED(CFG_CORE_RESERVED_SHM);
```

Comment:
could be `const`

Reply:
True but it would not make any practical difference (it's on the stack not global; I would definitely add `const` for a global variable).